roles/kubernetes/preinstall/tasks/0080-system-configurations.yml

```diff
@@ -136,3 +136,10 @@
     state: present
     reload: yes
   with_items: "{{ additional_sysctl }}"
+
+- name: Disable fapolicyd service
+  systemd:
+    name: fapolicyd
+    state: stopped
+    enabled: false
+  when: disable_fapolicyd
```

Comment:
So actually the CI error is related when fapolicyd is not installed this fail, you probably need a failed_when: false here

Reply:
Ok, I just added that line. Hope it fixes the CI error.